model/src/shared_protocol.rs

```diff
@@ -13,45 +13,40 @@ use maia_core::secp256k1_zkp::XOnlyPublicKey;
 use maia_core::PartyParams;
 use std::ops::RangeInclusive;
 
-pub(crate) async fn verify_cets(
+pub fn verify_cets(
     (oracle_pk, nonce_pks): (XOnlyPublicKey, Vec<XOnlyPublicKey>),
     counterparty: PartyParams,
     own_cets: Vec<(Transaction, EcdsaAdaptorSignature, interval::Digits)>,
     counterparty_cets: Vec<(RangeInclusive<u64>, EcdsaAdaptorSignature)>,
     commit_desc: Descriptor<bdk::bitcoin::PublicKey>,
     commit_amount: Amount,
 ) -> Result<()> {
-    tokio::task::spawn_blocking(move || {
-        for (tx, _, digits) in own_cets.iter() {
-            let counterparty_encsig = counterparty_cets
-                .iter()
-                .find_map(|(range, encsig)| (range == &digits.range()).then(|| encsig))
-                .with_context(|| {
-                    let range = digits.range();
+    for (tx, _, digits) in own_cets.iter() {
+        let counterparty_encsig = counterparty_cets
```

Comment:
Maybe it's not clear: I've moved it _outside_. The motivation was for the `model` library not to mandate that this computation has to be called from within `spawn_blocking`. And this is only relevant now because we just moved these methods from `daemon` to `model` in https://github.com/itchysats/itchysats/pull/2533/commits/3717fa29f1a78f1f9bc1da6fa95197bbc43362db.